components/Form/Select/Select.js

```diff
@@ -93,7 +93,7 @@ export default function Select({
    * @returns {string[]}
    */
   const getValueFromMulti = () => {
-    return options.filter(option => fieldValue.includes(option.value));
+    return options.filter(option => (fieldValue || []).includes(option.value));
```

Comment:
Resetting search params would basically causing this error.

<img width="1368" alt="Screen Shot 2020-10-12 at 5 22 36 PM" src="https://user-images.githubusercontent.com/17136024/95796255-6d2c4400-0cba-11eb-8e02-c8da8720509a.png">